ngApp/app.ts

```diff
@@ -1,6 +1,18 @@
 namespace codingskills {
 
-    angular.module('codingskills', ['ui.router', 'ngResource', 'ngMaterial']).config((
+    angular.module('codingskills', ['ui.router', 'ngResource', 'ngMaterial'])
+    
+    .constant('LEVELS', {

```

Comment:
consider creating a `const/const.ts` file and placing this in there.  this  seems like the wrong place to put this.